image_writers_guide/guidelines.adoc

```diff
@@ -5,143 +5,186 @@
+It is good practice to set environment variables with the `ENV` instruction. One example is to set the version of your project. This makes it easy for people find the version without looking at the [sysitem]#Dockerfile#. Another example is advertising a path on the system that could be used by another process, such as `JAVA_HOME`.
 
-=== Use EXPOSE to mark important ports
-See http://www.projectatomic.io/docs/docker-image-author-guidance/, subsection _Always `EXPOSE` Important Ports_
+*Avoiding Default Passwords*
 
-=== Set environment variables with ENV
-It’s a good idea to set environment variables with the `ENV` instruction. One example is to set version of your project. This will make it easy for people find the version without even looking at the Dockerfile.  Another example of an environment variable worth setting is when to advertise some path on the system that could be used by some process, like `JAVA_HOME`.
+Do not set default passwords. Many people will extend the image and forget to remove or change the default password. This can lead to security issues if there is a user with a well-known password in production. Passwords should be configurable using an environment variable instead. See the link:#using-env-vars[Using Environment Variables for Configuration] topic for more information.
 
-=== Do not set default passwords
-It is not a good idea to set default passwords. Many people will extend the image and forget to remove or change the default password. This will lead to security issues where in production there will be a user with a well-known passwords. Passwords should be configurable via environment variable instead (See anchor:configuration[])
+If you do choose to set a default password, ensure that an appropriate warning message is displayed when the container is started. The message should inform the user of the value of the default password and explain how to change it, such as what environment variable to set.
 
-If you do choose to set a default password, ensure that an appropriate warning message is displayed when the container is started.  The message should inform the user of the value of the default password and explaining how to change it (e.g. what environment variable to set).
+*Avoiding SSHD*
 
-=== Do not start (or install) SSHD
-Running SSHD should be avoided.  For accessing running containers you can use `docker exec` locally, or the OpenShift tooling which allows execution of arbitrary commands in running images.  Installing and running SSHD in your image opens up additional vectors for attack and requirements for security patching.
+Do not start or install SSHD in your image. For accessing running containers, you can use the `docker exec` command locally, or the OpenShift tooling which allows execution of arbitrary commands in running images. Installing and running SSHD in your image opens up additional vectors for attack and requirements for security patching.
 
```

Comment:
[Suggestion for readability]
Same as before, probably best to avoid starting a paragraph with "Do not....".

Do not start or install SSHD in your image. For accessing running containers, you can use the `docker exec` command locally, or the OpenShift tooling which allows execution of arbitrary commands in running images. Installing and running SSHD in your image opens up additional vectors for attack and requirements for security patching.  ==>> It is best to avoid running SSHD in your image. You can use the `docker exec` command locally to access containers that are running. Alternatively, you can use the OpenShift tooling since it allows you to execute arbitrary commands in images that are running. 

[Question]
What is the "OpenShift tooling"?